compiler/luci/import/src/CircleReader.cpp

```diff
@@ -151,6 +151,22 @@ MirrorPadMode luci_mirrorpad_mode(const circle::MirrorPadMode mode)
   return MirrorPadMode::UNDEFINED;
 }
 
+luci::CircleFullyConnected::WeightsFormat
+luci_weights_format(const circle::FullyConnectedOptionsWeightsFormat weights_format)
+{
+  switch (weights_format)
+  {
+    case circle::FullyConnectedOptionsWeightsFormat_DEFAULT:
+      return luci::CircleFullyConnected::WeightsFormat::DEFAULT;
+    case circle::FullyConnectedOptionsWeightsFormat_SHUFFLED4x16INT8:
+      return luci::CircleFullyConnected::WeightsFormat::SHUFFLED4x16INT8;
+    case circle::FullyConnectedOptionsWeightsFormat_SHUFFLED16x1FLOAT32:
+      return luci::CircleFullyConnected::WeightsFormat::SHUFFLED16x1FLOAT32;
+  }
+  assert(false);
+  return luci::CircleFullyConnected::WeightsFormat::UNDEFINED;
```

Comment:
By comment in IR, `UNDEFINED` is not permitted. How about throw an exception?

Reply:
From offline disussion;
- this was copied from exsiting other (optional) attributes
- this line itself is to check `set` argument
- it would be safe to throw for release build also to prevent invalid model file
- maybe sometime later we can revise existing codes